staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation.go

```diff
@@ -251,6 +252,26 @@ func (s *Validator) validateExpressions(ctx context.Context, fldPath *field.Path
 			continue
 		}
 		if evalResult != types.True {
+			tryMessage := true
+			if len(rule.MessageExpression) != 0 {
+				evalResult, evalDetails, err := compiled.MessageExpression.ContextEval(ctx, activation)
+				if evalDetails == nil {
+					klog.Warningf("runtime cost could not be calculated for message expression %q")
+				}
+				if err != nil {
+					klog.Warningf("error evaluating message expression, falling back: %v", err.Error())
```

Comment:
We should keep an eye on this. I'm a bit concerned someone will flood logs with this.

Reply:
An alternative would be to return a client warning..